README.md

```diff
@@ -26,6 +26,12 @@ Create an HPI file to install in Jenkins (HPI file will be in `target/slack.hpi`
 
     mvn package
 
+If you are having trouble getting mvn to package due to a test failure, please add this to your pom.xml
+
+	<properties>
+		<skipTests>true</skipTests>
+	</properties>
```

Comment:
I haven't tested this, yet.  But if it's failing unit tests as a result of the change I don't consider disabling unit tests a solution.